core/common/src/main/java/alluxio/util/IdUtils.java

```diff
@@ -91,4 +91,11 @@ public static long createSessionId() {
   public static long createMountId() {
     return getRandomNonNegativeLong();
   }
+
+  /**
+   * @return a random UUID which does not have hyphens
+   */
+  public static String createFileSystemContextId() {
```

Comment:
could this use a long instead? UUID is very long for something that shows up in metrics graphing UIs, and with the hostname included we only need the ID to be unique among clients on the same host. Even with 100k clients there's less than a 1 in a billion chance of collision.

Reply:
This UI is not displayed graph UI , but it's used for debugging only.
For now this ID is used to identifying a JVM on a node.

Reply:
I mean in graphana or wherever users choose to consume our metrics from.

If the ID is per-JVM per-node, that makes it even less necessary to have a UUID